### PR TITLE
Cdef work

### PR DESCRIPTION
**Pull request by xiphmont**
CDEF is working with no mismatches on testing.... much more work to do, but we can begin reviewing.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ DEC_FILE="dec_file.y4m"
 export RUST_BACKTRACE=1
 
 # Build and run encoder
-cargo run --bin rav1e --release -- $SEQ -o $ENC_FILE -s 3
+cargo run --bin rav1e --release -- $SEQ -o $ENC_FILE -s 3 -r rec_file.y4m
```
Comment (tdaede) [inline build.sh]: unrelated change (though maybe useful)

**Comment by tdaede** [comment]
I'll squash these to merge, unless you'd rather squash.

Have you tried an AWCY run? (or just looking at the images? does it look better?)

**Comment by xiphmont** [comment]
just looking at the images thusfar; they do in fact look better.

Go ahead and squash.

Monty
